examples/demo-react-native/scripts/bloatBundle.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+function bloatBundle() {
+  const bundlePath = './app.js';
+
+  try {
+    const debugBundle = fs.openSync(bundlePath, 'a');
+    try {
+      /// Bloating the bundle with a lot of data.
+      for (let i = 0; i < 25000; i++) {
```

Comment:
Some suggestions: 

* Extract to constants the number of lines to write and the bloat message
* Enable input args to allow overriding from outside vars here - number of lines to write, bundlepath